webpack.config.js

```diff
@@ -113,6 +113,9 @@ const wordpressExternals = ( context, request, callback ) =>
  */
 // eslint-disable-next-line no-unused-vars
 function getWebpackConfig( { externalizeWordPressPackages = false } = {}, argv ) {
+	const cssFilename =
+		isDevelopment || calypsoEnv === 'desktop' ? 'style-[name].css' : 'style-[name].[chunkhash].css';
```

Comment:
Do we necessarily need `style-` prefix?  The `.css` extension already tells us it's a stylesheet.

Reply:
We don't really need the prefix, it's just that our `.gitignore` file doesn't ignore all `/public/*.css` files, but only ones that match certain patterns: `/public/style*.css`. Easy to change, of course, but I choose the path of least resistance :)